### ISSUE TITLE
Compile: Add feature to continue where installation process left off after failure

### ISSUE THREAD

**Issue by ghost** [gobolinux/Compile#33]
It would be nice to have the option to tell `Compile` to continue the installation a package after it has been built successfully. Too many times a package like `Glibc` has too many `file exists` errors due to symlink conflicts from its previous installation. It like playing a game of Cat and Mouse where each time a silly failure like that occurs, I have to manually remove the symlink and rebuild again, hoping it will succeed the next time until it actually does. I have to rebuild `Glibc` over 10 times now because of this.

**Comment by Nuc1eoN** [comment]
I do not think it is (feasably) possible to continue installation at the exact install command of a file. We are simply running a spicy version of `make install` after all.

But it looks like Compile indeed lacks an option to skip (re)building and go straight to the install install step.

This should be a good thing to have.

**Comment by lucasvr** [comment]
> But it looks like Compile indeed lacks an option to skip the build process and go straight to the install install step.

`Compile -B` should do it.

**Comment by lucasvr** [comment]
> > `Compile` -B should do it.
> 
> It seems `Compile -B` only pulls the sources and does nothing more.

If the program at the source directory has been successfully compiled before, then `Compile -B` should proceed straight to `make install`.

**Comment by Nuc1eoN** [comment]
The `Compile --help` commmands states:
```
    -B, --no-build
        Do not build, just fetch the sources.
```
After I run `Compile LibXi` and let it build and install successfully, upon running `Compile -B` all I get is:
```
$ sudo Compile -B LibXi
Already up to date.
```
If I'd manually delete the tarball form /Data/Compile/Archives all I get is:
```
$ sudo Compile -B LibXi
Already up to date.
--2024-11-12 16:49:51--  https://www.x.org/releases/individual/lib/libXi-1.8.2.tar.xz
Resolving www.x.org (www.x.org)... 131.252.210.176
Connecting to www.x.org (www.x.org)|131.252.210.176|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 405200 (396K) [application/x-xz]
Saving to: 'libXi-1.8.2.tar.xz'

libXi-1.8.2.tar.xz            100%[===============================================>] 395.70K   547KB/s    in 0.7s    

2024-11-12 16:49:56 (547 KB/s) - 'libXi-1.8.2.tar.xz' saved [405200/405200]
```
It does not appear to attempt an installation.

**Comment by lucasvr** [comment]
Oh, my bad. The flag I wanted to mention is `-L/--lazy`. When used, it won't try to fetch the sources again. Also, it will reuse the existing unpacked source directory and won't call into `./configure` again. `make` and `make install` will run as usual -- so if the sources had been successfully compiled before, `make` should run rather quickly.
